packages/scripts/src/ManualRelay.js

```diff
@@ -59,7 +59,7 @@ async function main() {
     l2Web3,
     process.env.BRIDGE_DEPOSIT_ADDRESS || defaultDepositBoxMapping[Number(argv.chainId)],
     Number(argv.chainId),
-    latestL2BlockNumber - 9900,
+    latestL2BlockNumber - 99900,
```

Comment:
Is there somewhere where we should put this magic number so we don't have to remember it? Or is there at least a piece of documentation we can reference to note where it comes from?

Reply:
Yeah i'll add a comment linking to the inspiration code